iOS/LocalPackages/DataBrokerProtection-iOS/Sources/DataBrokerProtection-iOS/ManagingAndCoordinating/DataBrokerProtectionIOSManager.swift

```diff
@@ -110,6 +110,10 @@ public class DBPIOSInterface {
         func tryToFireWeeklyPixels()
     }
 
+    protocol WideEventsDelegate: AnyObject {
```

Comment:
What do you think about renaming this one to make it obvious that it's a PIR thing? Like:


```suggestion
    protocol DBPWideEventsDelegate: AnyObject {
```